examples/llama.py

```diff
@@ -180,15 +180,21 @@ def convert(name) -> Tensor:
     return lazy_tensors[0].cat(*lazy_tensors[1:], dim=axis)
   return {name: convert(name) for name in {name: None for model in models for name in model}}
 
-def load(fn:str):
+def load(fn: str):
   if fn.endswith('.index.json'):
     with open(fn) as fp: weight_map = json.load(fp)['weight_map']
     parts = {n: load(str(Path(fn).parent / Path(n).name)) for n in set(weight_map.values())}
     return {k: parts[n][k] for k, n in weight_map.items()}
   elif fn.endswith(".safetensors"):
     return safe_load(fn)
   else:
-    return torch_load(fn)
+    try: return torch_load(fn)
```

Comment:
Hmm, I don't love this try except. Can you show it always works? If it does, you def don't need the `elif` clause